index.js

```diff
@@ -1,5 +1,5 @@
 wrapAsync = (Meteor.wrapAsync)? Meteor.wrapAsync : Meteor._wrapAsync;
 Mongo.Collection.prototype.aggregate = function(pipelines) {
-  var coll = this._getCollection();
+  var coll = MongoInternals.defaultRemoteCollectionDriver().mongo.db.collection(this._name);
```

Comment:
Getting the collection from `defaultRemoteCollectionDriver` is wrong. It's possible to create a collection from another driver. We do this hack, because earlier we couldn't access the underline collection and db.

So, now we can get the collection directly from `this.rawCollection`. method. So use it.

And we need to make this backward compatible. If it's not there, simply use `_getCollection()`.